diff --git a/src/CMakeLists.txt b/src/CMakeLists.txt
index b10520e..5028a7f 100644
--- a/src/CMakeLists.txt
+++ b/src/CMakeLists.txt
@@ -27,11 +27,11 @@ TARGET_LINK_LIBRARIES(hibus ${libhibox} ${OPENSSL_LIBRARIES})
 
 ADD_EXECUTABLE(hibuscl cmdline.c drum_game.c libhibus.c helpers.c
         sign_verify_rsa_sha256.c sign_verify_hmac_sha256.c sign_verify_dummy.c)
-TARGET_LINK_LIBRARIES(hibuscl ${libhibox} ${OPENSSL_LIBRARIES} -lefence)
+TARGET_LINK_LIBRARIES(hibuscl ${libhibox} ${OPENSSL_LIBRARIES})
 
 ADD_EXECUTABLE(hibusd websocket.c unixsocket.c server.c endpoint.c builtin_endpoint.c
         helpers.c pattern_list.c sign_verify_rsa_sha256.c sign_verify_hmac_sha256.c sign_verify_dummy.c)
-TARGET_LINK_LIBRARIES(hibusd ${libhibox} ${libglib} ${OPENSSL_LIBRARIES} -lefence)
+TARGET_LINK_LIBRARIES(hibusd ${libhibox} ${libglib} ${OPENSSL_LIBRARIES})
 
 INSTALL(TARGETS hibus hibuscl
 	LIBRARY DESTINATION lib
diff --git a/src/builtin_endpoint.c b/src/builtin_endpoint.c
index 2c93983..ffcffbd 100644
--- a/src/builtin_endpoint.c
+++ b/src/builtin_endpoint.c
@@ -890,7 +890,8 @@ bool fire_system_event (BusServer* bus_srv, int bubble_type,
         }
         else {
             WSClient* wsc = (WSClient *)cause->entity.client;
-            strncpy (peer_info, wsc->remote_ip, sizeof (wsc->remote_ip));
+            size_t n = sizeof (wsc->remote_ip);
+            strncpy (peer_info, wsc->remote_ip, n);
         }
 
         n = snprintf (bubble_data, sizeof (bubble_data), 
