diff --git a/configure.ac b/configure.ac
index be22832a..e0800392 100644
--- a/configure.ac
+++ b/configure.ac
@@ -98,7 +98,6 @@ GLIB_MKENUMS=`$PKG_CONFIG --variable=glib_mkenums glib-2.0`
 AC_SUBST(GLIB_MKENUMS)
 
 PKG_CHECK_MODULES(LIBRSVG, \
-	gdk-pixbuf-2.0 >= $GDK_PIXBUF_REQUIRED \
 	glib-2.0 >= $GLIB_REQUIRED \
 	gio-2.0 >= $GIO_REQUIRED \
 	libxml-2.0 >= $LIBXML_REQUIRED \
@@ -113,10 +112,10 @@ PKG_CHECK_MODULES([GMODULE],[gmodule-2.0])
 
 case "$host" in
   *-*-mingw*)
-    PKG_CHECK_MODULES([RSVG_CONVERT],[gio-2.0 gio-windows-2.0 gdk-pixbuf-2.0 cairo pangocairo])
+    PKG_CHECK_MODULES([RSVG_CONVERT],[gio-2.0 gio-windows-2.0 cairo pangocairo])
     ;;
   *)
-    PKG_CHECK_MODULES([RSVG_CONVERT],[gio-2.0 gio-unix-2.0 gdk-pixbuf-2.0 cairo pangocairo])
+    PKG_CHECK_MODULES([RSVG_CONVERT],[gio-2.0 gio-unix-2.0 cairo pangocairo])
     ;;
 esac
 
